announcement/announcement-impl/impl/src/java/org/sakaiproject/announcement/impl/BaseAnnouncementService.java

```diff
@@ -1271,15 +1272,20 @@ private List<String> getExcludedSitesFromTabs() {
 	    return l;
 	}
 
+	public Filter getMaxAgeInDaysFilter(int maxAgeInDays) {
+
+		ViewableFilter viewableFilter = new ViewableFilter(null, null, Integer.MAX_VALUE, this);
```

Comment:
this is a lot of announcements! What's a practical value? 1000?

Reply:
@ottenhoff How's that look now?

Reply:
thanks!